lib/monthly.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
```

Comment:
Avoid using vague terms like 'Prefillable' in the module name. Consider renaming the module to more accurately describe the service's purpose.